### PR TITLE
Fix unittests

### PR DESCRIPTION
**Pull request by rchl**

### CODE REVIEW

**Review by rwols — COMMENTED** [review]
I think the `skipIf` in test_views.py can now also be removed.

File: tests/test_views.py [hunk 1]
```diff
@@ -192,7 +190,7 @@ def test_minihtml_format_string(self) -> None:
 
     def test_minihtml_format_marked_string(self) -> None:
         content = "<div>text\n</div>"
-        expect = "<div>text</div>"
+        expect = "<div>text\n</div>"
```
Comment (rwols) [inline tests/test_views.py:193]: Is this due to a change in mdpopups?

**Review by rwols — APPROVED** [review]
Yep, that's pretty much all the tests I broke. Nice work.

File: tests/test_views.py (hunk 1 above)
Comment (rchl) [reply tests/test_views.py:193]: Yes.
I thought it was supposed to convert newlines to spaces when `nl2br` is disabled but it seems like it keeps newline here for some reason. Maybe has something to do with our frontmatter config:
```py
        frontmatter = {
            "allow_code_wrap": True,
            "markdown_extensions": [
                {
                    "pymdownx.escapeall": {
                        "hardbreak": False,
                        "nbsp": False
                    }
                }
            ]
        }
```

But to be fair, I don't really care as whitespace doesn't change anything in block elements so there won't be any visual change.

Comment (rchl) [reply tests/test_views.py:193]: All in all, a newline will render as space in block on inline element when rendered in HTML so as expected per the mdpopups fix.

**Pull request merged**